### PR TITLE
PL 148 Features display on welcome page

### PR DESCRIPTION
**Pull request by gabrielekarra**
New welcome page that display features

### CODE REVIEW

**Review by matteovisotto — COMMENTED** [review]
Very well done, just two things:
- The card shadow is inside the slider so not visible and it is good to see it.
- The current card dot is not highlighted, also pagination dots have to indicate the current one.

**Comment by paolocorsa** [comment]
I agree with Matteo, the important thing for me is to add small dots to change card when you want and don't wait.

**Comment by gabrielekarra** [comment]
> Very well done, just two things:
> 
> * The card shadow is inside the slider so not visible and it is good to see it.
> * The current card dot is not highlighted, also pagination dots have to indicate the current one.

fixed, ty

**Comment by gabrielekarra** [comment]
> I agree with Matteo, the important thing for me is to add small dots to change card when you want and don't wait. Image and text are perfect!

done, ty

File: src/templates/index.html [hunk 1]
```diff
@@ -50,7 +50,77 @@ <h1 class="font-header better"> BETTER </h1>
...
+                                <div class="card">
+                                    <img src="assets/highligh.png" class="icon-welcome-page" />
+                                    <div class="container">
+                                        <h4><b>Highligh commonly used, topic-specific phrasings
+                                            </b></h4>
+                                        <p class="text-card-welcome-page">A paper can involve using many typical
+                                            phrases/sentences
+                                            for introduction, background description and experiment result analyses.
+                                            Identifying and
+                                            highlighting these sentences
```
Comment (tonesija) [inline src/templates/index.html:109]: This is not true, we only highlight the keywords. Not phrases or sentences. We could hightlight the whole sentence however.
We have to decide about that.

**Review by tonesija — COMMENTED** [review]
Problems:
- searchs.png should be called searches.png or search.png.
- highlight.png*
- I can't see the dots when the page opens
- 1st and thirds dot are the same -> there is no knowledge graph one

**Comment by gabrielekarra** [comment]
@tonesija fixed now, ty

**Comment by matteovisotto** [comment]
I think it is on now, +1

**Review by tonesija — COMMENTED** [review]
Only 3 things:
- I thinks it's better to say Generate Knowledge graph then Make your knowledge graph
- I don't like 'your', because the knowledge graph, cross referneces and summaries do not belong to the user, It would be better to write Generate Knowledge graph, Summarize the paper and Explore paper and Cross-Refs
- Dots are now visible when opened, but the first one is not green, a dot only becomes green/active if you click on it

**Comment by gabrielekarra** [comment]
> 

fixed them all

**Comment by tonesija** [comment]
+1

**Pull request merged**